### ISSUE TITLE
one-line: check-whitespace not working

### ISSUE THREAD

**Issue by yubaoquan** [palantir/tslint#3372]
### Bug Report

- __TSLint version__: 1.9.0
- __TypeScript version__: 2.6
- __Running TSLint via__: Atom

#### TypeScript code being linted

```ts
if (e.code == 'EADDRINUSE') {
    console.log(`\nPort ${config.serverConfig.port} is in use, please check.`)
    server.close()
    process.exit()
}else {
    //
}
```

with `tslint.json` configuration:

```json
{
    "extends": [
        "tslint-eslint-rules"
    ],
    "rules": {
        "trailing-comma": [true, {"multiline": "always", "singleline": "never"}],
        "no-extra-semi": true,
        "semicolon": [true, "never"],
        "whitespace": [
            true,
            "check-branch",
            "check-decl",
            "check-operator",
            "check-module",
            "check-separator",
            "check-rest-spread",
            "check-type",
            "check-typecast",
            "check-type-operator",
            "check-preblock"
        ],
        "no-trailing-whitespace": true,
        "one-line": [
            true,
            "check-open-brace",
            "check-else",
            "check-catch",
            "check-finally",
            "check-whitespace"
        ]
    }
}

```

#### Actual behavior

Missing preceding whitespace of `else`, no error.

#### Expected behavior 

Missing preceding whitespace of `else`, report an error.

**Comment by ajafff** [comment]
I doubt 1.9.0 is your currently installed version of TSLint.
Can you try updating to 5.8.0? The new version definitely has this check.

**Comment by yubaoquan** [comment]
Resolved thank you
